cpp/src/aztec/proof_system/types/polynomial_manifest.hpp

```diff
@@ -193,7 +194,8 @@ static constexpr PolynomialDescriptor standard_honk_polynomial_manifest[STANDARD
     PolynomialDescriptor("ID_2", "id_2_lagrange", false, false, PERMUTATION, ID_2),                  //
     PolynomialDescriptor("ID_3", "id_3_lagrange", true, false, PERMUTATION, ID_3),                   //
     PolynomialDescriptor("LAGRANGE_FIRST", "L_first_lagrange", false, false, OTHER, LAGRANGE_FIRST), //
-    PolynomialDescriptor("LAGRANGE_LAST", "L_last_lagrange", false, false, OTHER, LAGRANGE_LAST)     //
+    PolynomialDescriptor("LAGRANGE_LAST", "L_last_lagrange", false, false, OTHER, LAGRANGE_LAST),    //
+    PolynomialDescriptor("POW_ZETA", "pow_zeta", false, false, WITNESS, POW_ZETA)                    //
```

Comment:
Any particular reason why this is a WITNESS? Maybe this is all temporary and it doesn't matter?

Reply:
Cf the previous comment. It can't be computed without `zeta`, which depends on the witnesses.